### PR TITLE
docs: update documentation for api_reference

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates documentation for api_reference**

### CODE REVIEW

**Comment by Marketionist** [comment]
@aslakhellesoy @charlierudolph please review

File: docs/support_files/api_reference.md [hunk 1]
```diff
@@ -125,8 +125,8 @@ setDefinitionFunctionWrapper(function(fn, options) {
         // call a method on world
         this.doScreenshot();
         // rethrow error to avoid swallowing failure
-        throw error;
-      })  
+        throw error
```
Comment (charlierudolph) [inline docs/support_files/api_reference.md]: Can we add semicolons to this example where appropriate. I think i prefer having them (and will be updating cucumber-js source to have them soon)

**Comment by aslakhellesoy** [comment]
I’d prefer to use a consistent style everywhere. The monorepo does not use semicolons in TypeScript code, and we use `ts-lint —fix` with the `prettier` plugin to automatically format code.

**Comment by charlierudolph** [comment]
Cucumber-js is using eslint + prettier and I was leaning towards using the default configuration of prettier which includes semicolons

File: docs/support_files/api_reference.md (hunk 1 above)
Comment (Marketionist) [reply docs/support_files/api_reference.md]: Sure, I also prefer semicolons - will add them if everybody agrees

**Comment by Marketionist** [comment]
@aslakhellesoy @charlierudolph please re-review

**Comment by Marketionist** [comment]
@aslakhellesoy I can merge this PR if you are ok with it?

**Comment by aslakhellesoy** [comment]
Ok go ahead and merge. We might change to using the default setting in the monorepo too, if that’s what the majority prefers. Personally I prefer no semis but I’m not religious about it.

**Pull request merged**